packages/vscode-extension/lib/expo_router_plugin.js

```diff
@@ -1,18 +1,21 @@
-import { useSyncExternalStore, useEffect } from "react";
-import { useRouter } from "expo-router";
 import { store } from "expo-router/build/global-state/router-store.js";
```

Comment:
Moved the store import to the top so it's more evident that those two files are different only there.